pages/index.js

```diff
@@ -1,3 +1,22 @@
+import CardComponent from "@/components/cardcomponent/CardComponent";
+import { initialData } from "@/helpers/data/data";
```

Comment:
Instead of importing your data here again you could also hand it over from your `_app.js` to your index page. :)